ajax/libs/vega/package.json

```diff
@@ -13,8 +13,13 @@
     {
       "basePath": "",
       "files": [
-        "vega.js",
-        "vega.min.js"
+        "vega*.+(js|js.map)"
+      ]
```

Comment:
for v3, I think maybe we can just remove this filemap with `"basePath": ""` since there is no release file at root, am I right?